ui/src/app/login/components/login.scss

```diff
@@ -9,7 +9,7 @@ $logoNegativePath: 'assets/images/argo.png';
     border-bottom: 2px solid $argo-color-gray-3;
 }
 
-.login {
+:not(.application-details).login {
```

Comment:
I think the proper fix will be to prefix the class name that's injected based on the application name.

Reply:
Is a proposal required?